content/en/opentelemetry/guide: update ingestion sampling guide (#17323)
DataDog/documentation: content/en/opentelemetry/guide/ingestion_sampling_with_opentelemetry.md

### Collector-level sampling

At the OpenTelemetry collector level, you can do _tail-based sampling_, which allows you to define more advanced rules to keep accrued visibility over error or high latency traces.

The [Tail Sampling Processor][8] and [Probabilistic Sampling Processor][9] allow you to sample traces based on a set of rules at the collector level.

**Note**: Tail sampling's main limitation is that all spans for a given trace must be received by the same collector instance for effective sampling decisions. If the trace is distributed across multiple collector instances, there’s a risk that some parts of a trace are dropped whereas some other parts of the same trace are sent to Datadog.

To ensure that APM metrics are computed based on 100% of the applications' traffic while using collector-level tail-based sampling, preprend the [Datadog Processor][10] in front of your sampling processor in the collectors' traces pipeline. The processor is available with OpenTelemetry Collector Contrib v0.69.0+.

See the [ingestion volume control guide][7] for information about the implications of setting up trace sampling on trace analytics monitors and metrics from spans.

> ### Sampling with the Datadog Agent
> 
> When using the OTLP Ingest in the Datadog Agent, a probabilistic sampler [is available][11] starting with the Datadog Agent version 7.44.0. It can be configured by using the DD_OTLP_CONFIG_TRACES_PROBABILISTIC_SAMPLER_SAMPLING_PERCENTAGE environment variable, or by using the following YAML in your agent's configuration file:
> 
> ```yaml
> otlp_config:
>   # ...
>   traces:
>     probabilistic_sampler:
>       sampling_percentage: 50
> ```
> 
> The above example will sample only 50% of traces.
> 
> If the user sets any sampling priorities, the probabilistic sampler will follow those choices. Additionally, errors may be sampled by the agent's [error sampler][12] depending on its configuration. The [rare sampler][13] may also be used in conjunction with the probabilistic sampler if desired.

Reviewer: ```suggestion
### Sampling with the Datadog Agent

When using the Datadog Agent [OTLP Ingest][2], a [probabilistic sampler][11] is available starting from the version 7.44.0 of the Datadog Agent version. Use the `DD_OTLP_CONFIG_TRACES_PROBABILISTIC_SAMPLER_SAMPLING_PERCENTAGE` environment variable to configure it, or set the following YAML in your agent's configuration file:

```yaml
otlp_config:
  # ...
  traces:
    probabilistic_sampler:
      sampling_percentage: 50
\```

In the above example, 50% of traces will be captured.

**Note**: Probabilistic sampler properties ensure that only complete traces are ingested if you use the same sampling percentage across all agents.

The probabilistic sampler will ignore spans for which the sampling priority was already set at the SDK level.
Additionally, spans not caught by the probabilistic sampler might still be captured by the Datadog Agent's [error and rare samplers][12], in order to ensure a higher representation of errors and rare endpoint traces in the ingested dataset. 
```

## Monitor ingested volumes from Datadog UI

> You can leverage the [APM Estimated Usage dashboard][14] and the estimated usage metric `datadog.estimated_usage.apm.ingested_bytes` to get visibility into your ingested volumes for a specific time period. Filter the dashboard to specific environments and services to see which services are responsible for the largest shares of the ingested volume.

Reviewer: ```suggestion
You can leverage the [APM Estimated Usage dashboard][13] and the estimated usage metric `datadog.estimated_usage.apm.ingested_bytes` to get visibility into your ingested volumes for a specific time period. Filter the dashboard to specific environments and services to see which services are responsible for the largest shares of the ingested volume.
```

## Further Reading

{{< partial name="whats-next/whats-next.html" >}}

[2]:/opentelemetry/otel_collector_datadog_exporter/?tab=alongsidetheagent#5-run-the-collector
[1]: /opentelemetry/otel_collector_datadog_exporter
[2]: /opentelemetry/otlp_ingest_in_the_agent
[3]: /tracing/metrics/metrics_namespace/
[4]: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#traceidratiobased
[5]: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#parentbased
[6]: /dashboards/functions/#add-a-function
[7]: /tracing/guide/trace_ingestion_volume_control/#effects-of-reducing-trace-ingestion-volume
[8]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md
[9]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/probabilisticsamplerprocessor/README.md
[10]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/datadogprocessor
[11]: https://github.com/DataDog/datadog-agent/blob/fd550c6/pkg/config/config_template.yaml#L3533-L3545

> [12]: https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/?tab=java#error-traces
> [13]: https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/?tab=java#rare-traces

Reviewer: ```suggestion
[12]: /tracing/trace_pipeline/ingestion_mechanisms/#error-and-rare-traces
```

> [14]: https://app.datadoghq.com/dash/integration/apm_estimated_usage

Reviewer: ```suggestion
[13]: https://app.datadoghq.com/dash/integration/apm_estimated_usage
```
